src/error.ts

```diff
@@ -7,26 +7,55 @@ export type AnyError = MongoError | Error;
 
 const kErrorLabels = Symbol('errorLabels');
 
+/** @internal MongoDB Error Codes */
+export const MONGODB_ERROR_CODES = Object.freeze({
```

Comment:
Since we're using codes in a number of places it seemed to make sense to make an enum out of them all. Luckily the git diff shows the naming is the same as the comment that was placed next to a lot of the numbers.